cmd/scollector/collectors/snmp_cisco.go

```diff
@@ -2,66 +2,48 @@ package collectors
 
 import (
 	"fmt"
-	"math/big"
 	"time"
 
-	"bosun.org/metadata"
+	"bosun.org/cmd/scollector/conf"
 	"bosun.org/opentsdb"
 )
 
-const (
-	ciscoCPU     = ".1.3.6.1.4.1.9.9.109.1.1.1.1.6"
-	ciscoMemFree = ".1.3.6.1.4.1.9.9.48.1.1.1.6"
-	ciscoMemName = ".1.3.6.1.4.1.9.9.48.1.1.1.2"
-	ciscoMemUsed = ".1.3.6.1.4.1.9.9.48.1.1.1.5"
-)
-
 // SNMPCisco registers a SNMP CISCO collector for the given community and host.
-func SNMPCisco(community, host string) {
+func SNMPCisco(cfg conf.SNMP) {
+
+	mib := conf.MIB{}
```

Comment:
I would combine all the .BaseOid and .Metrics and .Trees setting into one giant init because I think it's easier to read. Or not if you think this is better.